[DOCU-2402] Remove KM and decK, and HTTPie instructions from comprehensive docs. (#4137)
Kong/docs.konghq.com: src/gateway/get-started/comprehensive/protect-services.md

## Why Use Rate Limiting?

Rate limiting protects the APIs from accidental or malicious overuse. Without rate limiting, each user may request as often as they like, which can lead to spikes of requests that starve other consumers. After rate limiting is enabled, API calls are limited to a fixed number of requests per second.
## Set up Rate Limiting

{:.note}

Call the Admin API on port `8001` and configure plugins to enable a limit of five (5) requests per minute, stored locally and in-memory, on the node.

```sh

> curl -i -X POST http://<admin-hostname>:8001/plugins \

Reviewer: ```suggestion
curl -i -X POST http://localhost:8001/plugins \
```

> curl -i -X GET http://<admin-hostname>:8000/mock/request

Reviewer: ```suggestion
curl -i -X GET http://localhost:8000/mock/request
```

```
{% endnavtab %}

{% navtab Web Browser %}

> 1. Enter `<admin-hostname>:8000/mock` and refresh your browser six times.

Reviewer: ```suggestion
1. Enter `localhost:8000/mock` and refresh your browser six times.
```

>     After the 6th request, you’ll receive an error message.

Reviewer: ```suggestion
    After the sixth request, you’ll receive an error message.
```

2. Wait at least 30 seconds and try again.

>     The service will be accessible until the sixth (6th) access attempt within a 30-second window.

Reviewer: ```suggestion
    The service will be accessible until the sixth access attempt within a 30-second window.
```
